apps/console/src/extensions/components/branding/components/preview/branding-preference-preview.tsx

```diff
@@ -36,6 +44,8 @@ import { EmailTemplateScreenSkeleton } from "./email-template-screen-skeleton";
 import { LoginScreenSkeleton } from "./login-screen-skeleton";
 import { MyAccountScreenSkeleton } from "./my-account-screen-skeleton";
 import { AppState } from "../../../../../features/core/store";
+import { ReactComponent as CustomLayoutSuccessImg } from
+    "../../../../../themes/wso2is/assets/images/branding/custom-layout-success.svg";
```

Comment:
We should not hardcode the theme names, instead we should import from the themes module

Reply:
Currently I couldn't found a way to include the images without hardcoding the theme. There are some other places as well we need to fix this. https://github.com/wso2/identity-apps/blob/8e833d1848e906f28aae8265bc606156f1be1454/apps/console/src/features/core/components/header.tsx#L48-L56